third_party/move/move-vm/profiler/README.md

```diff
@@ -0,0 +1,16 @@
+# VM Profiler
+
+This is a lightweight Rust profiling utility for instrumenting function and instruction execution inside the Move VM.
+It provides per-function and per-instruction timing via RAII guards
+and the [usdt](https://docs.rs/usdt) crate.
+
+---
+
+## Usage
+
+To use this profiler, attach an external tracing tool (e.g. `dtrace` on macOS or `bpftrace` on Linux).
```

Comment:
We should probably also mention that the binary should be built with probes feature enabled?